WordPress/gutenberg
# Separator Border Block Support

## What?
Add border block support to the Separator block.

Part of https://github.com/WordPress/gutenberg/issues/43247

## Why?

`Separator` block is missing border support.

## How?
Adds the border block support in block.json

## Testing Instructions

- Go to Global Styles setting ( under appearance > editor > styles > edit styles > blocks )
- Make sure that  Separator block's border is configurable via Global Styles
- Verify that Global Styles are applied correctly in the editor and frontend
- Edit template/page, Add  Separator block and Apply the border styles
- Verify that block styles take precedence over global styles
- Verify that block borders display correctly in both the editor and frontend

### Testing Instructions for Keyboard
None

## Screenshots or screencast 
In Backend:

![separator_backend](https://github.com/user-attachments/assets/cf68c4cc-2583-49ce-b090-60db1f6de2b7)


In Frontend:

![separator_frontend](https://github.com/user-attachments/assets/292d10d2-ffc7-4881-a5e2-8022dfdf891a)
